src/tiled/mainwindow.cpp

```diff
@@ -99,6 +102,95 @@ using namespace Tiled::Internal;
 using namespace Tiled::Utils;
 
 
+namespace {
+
+template <typename Format>
+struct ExportDetails
+{
+   Format* mFormat;
```

Comment:
Coding style: `*` should hug the variable name.

It's easy to remember if you realize this is how C++ associates the pointer: `int a = 3, *b = &a;`

Reply:
Done

I've been writing C++ with the opposite left adjusted style e.g. ``int* number`` for many years so it's just a matter of habit. Happy to conform to your code style so thanks for picking it up :)